lib/express_validator.js

```diff
@@ -441,6 +441,10 @@ function validateSchema(schema, req, loc, options) {
       if (opts != null && !Array.isArray(opts)) {
         opts = [opts];
       }
+      
```

Comment:
There's an extra trailing spaces here causing the tests to fail. Please remove them